__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -659,3 +643,78 @@ describe('TaskDependency', () => {
         });
     });
 });
+
+jest.mock('next/router', () => ({
+    useRouter() {
+        return {
+            query: {
+                dev: 'true',
+            },
+            push: mockNavigateToUpdateProgressPage,
+        };
+    },
+}));
+
+describe('Details component', () => {
+    it('Renders the detail type and value correctly', () => {
+        renderWithRouter(<Details detailType="Assignee" value="John Doe" />);
+        const detailType = screen.getByText('Assignee:');
+        const detailValue = screen.getByText('John Doe');
+        expect(detailType).toBeInTheDocument();
+        expect(detailValue).toBeInTheDocument();
+    });
+
+    it('Renders a GitHub link correctly', () => {
+        const githubIssueUrl = 'https://github.com/example/repo/issues/123';
```

Comment:
can we have this fixtures in fixtures file?

Reply:
Didn't fully understand what are you trying to say ?
Also, I didn't find any folder named fixtures

Reply:
It's present id this folder
website-status_mocks_\db

Reply:
If i'm moving all of this files to mocks, other test are failing. I will do this in the next PR